### PR TITLE
add slide with information about why we use set.seed()

### PR DESCRIPTION
**Pull request by EmilHvitfeldt**
To close https://github.com/tidymodels/workshops/issues/110

I put it here, since it is right after we first show `set.seed()`

### CODE REVIEW

**Review by hfrick — COMMENTED** [review]
Should we have a bridge-sentence which uses "pseudo-random" to explain that while the numbers are made to behave like random numbers, their generation is deterministic given a "seed"?

I'm pretty sure we end all sentences on the slides with a `.`, could you add those here as well, please?

File: slides/intro-02-data-budget.qmd [hunk 1]
```diff
@@ -192,6 +192,14 @@ This function uses a good default, but this depends on your specific goal/data
 We will talk about more powerful ways of splitting, like stratification, later
 :::
 
+## What is `set.seed()`?
+
+Creating a split in the data requires some random number generation to determine which observations go in which set
+
+This random number generation is deterministic in R, and we can **seed** it to get the same results over and over again
+
+The seed you pick doesn't matter, as long as you don't try a bunch of seeds and pick the one that gives you the best performance
```
Comment (hfrick) [inline slides/intro-02-data-budget.qmd:201]: 🤍

**Comment by EmilHvitfeldt** [comment]
> Should we have a bridge-sentence which uses "pseudo-random" to explain that while the numbers are made to behave like random numbers, their generation is deterministic given a "seed"?

It isn't a bad idea, but I'm having a hard time fitting it on the slides in a natural way. Do you mind taking a stab at it?

**Comment by hfrick** [comment]
What do you think? Feel free to tweak further if you want!

**Review by hfrick — APPROVED** [review]

**Comment by EmilHvitfeldt** [comment]
I love it!

**Pull request merged**